tools/build_node.js

```diff
@@ -173,8 +173,9 @@ async function buildNode(options) {
     if (stat.isDirectory()) return;
 
     if (file.endsWith(".css")) {
-      installCleanCSS(`./src/styles/${file}`, `styles/${file}`);
-      installCleanCSS(`./src/styles/${file}`, `scss/${file.replace(".css", ".scss")}`);
+      installCleanCSS(`./src/styles/${file}`, `styles/${file}`, { minify: false });
```

Comment:
This is slightly breaking though, as the content of `.css` here is changing... (from minified to un-minified). I think this is small enough break that it could be addressed in changelog with something like:

```
CAVEATS
- In the `node` build `styles/*.css` files now ship un-minified
- Minified counterparts can be found at `styles/*.min.css` (consistent with our CDN packaging)
```

Does that read clearly to you?  If so I think we may be ready to ship this one. :)

Reply:
Yes, maybe mention "\`node\`/\`cdn\` builds" (`node`/`cdn` builds are basically the same).

Would you land these docs changes or do you want me to to push them?

PS: also `??` (from https://github.com/highlightjs/highlight.js/pull/3807#discussion_r1233529800) needs to be changed according to the linter

Reply:
I'll handle the docs and lint issue, I'll try to get to this in next few days.

Reply:
@joshgoebel didn't you forget?

Reply:
@joshgoebel any news?